test/helpers/URMHandlerBase.sol

```diff
@@ -0,0 +1,467 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.30;
+
+// External Imports
+import {Test} from "forge-std/Test.sol";
+import {console} from "forge-std/console.sol";
+
+// Internal Imports
+import {FakeProtocolContract} from "test/fakes/FakeProtocolContract.sol";
+import {RollbackSet, LibRollbackSet, RollbackProposal} from "test/helpers/RollbackSet.sol";
+import {RollbackTransactionGenerator} from "test/helpers/RollbackTransactionGenerator.sol";
+import {IGovernor} from "@openzeppelin/contracts/governance/IGovernor.sol";
+import {URMCore} from "src/contracts/URMCore.sol";
+
+/// @title URMHandlerBase
+/// @notice Base handler contract for URM invariant testing
+/// @dev This abstract class contains all the common functionality between
+///      URMCompoundManager and URMOZManager handlers. Subclasses need to implement:
+///      - _getURM() - to return the URM contract instance
+///      - _getDescription() - to return the description string
+abstract contract URMHandlerBase is Test {
+  using LibRollbackSet for RollbackSet;
+
+  FakeProtocolContract[] public targets;
+  RollbackSet public _rollbackSet;
+  bytes4[] public selectors;
```

Comment:
nit: some more disambiguation requested: in an invariant/handler context, "selectors" can often mean the selectors that the call fuzzer is targeting. but here we're talking about inputs for generation of proposal calldata.

Reply:
Fixed at https://github.com/ScopeLift/governance-rollback/pull/59/commits/b24fbee595a5f1903cfe2f6b6ad3a730fb650bf9